openmc/deplete/results.py

```diff
@@ -0,0 +1,496 @@
+"""The results module.
+
+Contains results generation and saving capabilities.
+"""
+
+from collections import OrderedDict
+import copy
+from warnings import warn
+
+import numpy as np
+import h5py
+
+from . import comm, have_mpi, MPI
+from .reaction_rates import ReactionRates
+
+_VERSION_RESULTS = (1, 0)
```

Comment:
If this is exported for `ResultsList` we shouldnt keep it as a private constant. `VERSION_RESULTS` instead of `_VERSION_RESULTS` would be better